### ISSUE TITLE
Is there a way to map a client provided path to another?

### ISSUE THREAD

**Issue by caesarsamsi** [Kozea/Radicale#410]
Mac's Yosemite Contacts doesn't offer editing of the server path for CardDav accounts. We can only provide:

- User Name
- Password
- Server Address

Whatever is in the Server Address (even if I provide https://myserver:5232/me@me.com/contacts.vcf/)
Is chopped up and only the name of the server (i.e. myserver without the port number) survives as the server address.
Only the first segment of the full server path (i.e. the bit about the user from say /testuser/contacts.vcf)

Basically Apple Contacts uses that first segment to query for an address book in radicale. There is a need to map that first segment into the real server path. So from /testuser/ map to /testuser/contacts.vcf/

Help?

**Comment by caesarsamsi** [comment]
![apple contacts](https://cloud.githubusercontent.com/assets/8953119/15457878/10fa4344-2047-11e6-8471-86d7eab631b3.jpg)

**Comment by caesarsamsi** [comment]
Found a workaround. Apple's Contacts app default port is 8443 with SSL.

So I changed the server to listed to 8443, and _then_ I could specify a complete server path such as:
https://myserver/myname@somewhere.com/contacts.vcf.

Worked like a charm.
